stocks/go/go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/go-flutter-desktop/examples/stocks/desktop
+module stocks/go
```

Comment:
Just wondering why the prefix was removed?

Reply:
Oh I just re inited the stocks app, but forgot to provide the module. Will fix that